config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   root 'pages#index'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/signup',  to: 'users#new'
+  post '/signup', to: 'users#create'
```

Comment:
Why do we need this route? We have already the route define for POST create new user.

Reply:
removed ```post '/signup', to: 'users#create'```
using default create controller from resources